src/t8_forest/t8_forest.c

```diff
@@ -450,12 +456,15 @@ t8_forest_commit (t8_forest_t forest)
     mpiret = sc_MPI_Comm_rank (forest->mpicomm, &forest->mpirank);
     SC_CHECK_MPI (mpiret);
     /* Compute the maximum allowed refinement level */
+    t8_debugf ("compute maxlevel of forest:\n");
     t8_forest_compute_maxlevel (forest);
+    t8_debugf ("computed maxlevel: %i\n", forest->maxlevel);
     T8_ASSERT (forest->set_level <= forest->maxlevel);
     /* populate a new forest with tree and quadrant objects */
     if (t8_forest_refines_irregular (forest) && forest->set_level > 0) {
       /* On root level we will also use the normal algorithm */
-      t8_forest_populate_irregular (forest);
+      t8_forest_populate_irregular (forest, 1);
+      //      t8_forest_populate_irregular (forest, !forest->set_initial_partition_according_to_cmesh);
```

Comment:
```suggestion
```

Reply:
TODO: investigate further, because it looks like the second argument is not even used in `t8_forest_populate_irregular`